src/constants/data.ts

```diff
@@ -28,29 +29,29 @@ export interface ExerciseType {
 export const EXERCISES: ExerciseType[] = [
   {
     key: ExerciseKeys.vocabularyOverview,
-    title: 'Vokabelübersicht',
-    description: 'Alle Wörte anschauen',
+    title: labels.exercises.overview.title,
+    description: labels.exercises.overview.description,
```

Comment:
I think we should be consistent with naming of the exercises in all constants, labels and files... I am really confused which exercise is which :see_no_evil:

Reply:
yes, there is a ticket for this.